runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json

```
@@ -385,6 +403,18 @@
       "name": "Ruby Amulet (U)",
       "xp": 85
     },
+    {
+      "level": 51,
+      "icon": 6324,
+      "name": "Snakeskin chaps",
+      "xp": 50
+    },
+    {
+      "level": 53,
+      "icon": 18725,
```

https://github.com/runelite/runelite/blob/master/runelite-api/src/main/java/net/runelite/api/ItemID.java#L3780
```suggestion
      "icon": 6322,
```